build.sbt

```diff
@@ -520,32 +520,6 @@ lazy val `http4s-swagger` = projectMatrix
   )
   .http4sJvmPlatform(allJvmScalaVersions, jvmDimSettings)
 
-lazy val `doc-check` = projectMatrix
```

Comment:
I crap, I forgot to remove my test module loll

Reply:
no worries :)